modules/analytics/src/main/java/com/intuit/wasabi/analytics/impl/ExperimentDetailsImpl.java

```diff
@@ -89,7 +89,7 @@ public ExperimentDetailsImpl(@CassandraRepository ExperimentRepository cassandra
      * @return the same ExperimentDetail object but with additional information
      */
     private ExperimentDetail getBucketData(ExperimentDetail exp) {
-        List<Bucket> buckList = buckets.getBuckets(exp.getId()).getBuckets();
+        List<Bucket> buckList = buckets.getBuckets(exp.getId(), false).getBuckets();
```

Comment:
@mans4singh it's internal interface. is it really necessary? having an additional method in Buckets Interface will affect both CassandraExperimentRepository and DatabaseExperimentRepository (MySQL).